### ISSUE TITLE
Linting Action Always Fails

### ISSUE THREAD

**Issue by WildJimmy** [ubsuny/ML_LCVR-CP2P2024#3]
The linting action always fails. Kind of annoying and also means that it's probably not working properly which defeats the purpose anyhow.

**Comment by reshnashrestha** [comment]
Yeah, it is kind of annoying just because pylint suggests unnecessary stuffs as problems to be fixed, which are in fact necessary.

**Comment by Mosaddeq107** [comment]
Hi @WildJimmy, your workflow failed because you didn't add a Python file to your repository. 
The failed message indicates that your Pylint code ("*.py") will search for the files containing the .py extension. Since it didn't find any Python files, it failed always. 

![screenshot__ 2024-02-23 at 11 51 38 AM](https://github.com/ubsuny/ML_LCVR-CP2P2024/assets/66480280/ecefc220-4146-4ee5-a300-0c0a3f9fa6dd)

Once you add a Python file, it will work. I forked your repo, added a sample Python file, and made Linting work. 

![screenshot__ 2024-02-23 at 11 58 58 AM](https://github.com/ubsuny/ML_LCVR-CP2P2024/assets/66480280/4f125f45-71dd-4676-a289-6e8f7304c284)

**Comment by laserlab** [comment]
Looks to me that now only normal lifting issues are reported mostly in the web app. You have a score of 3.41 with most things easy to fix. There is also a vscode plugin that lints it for you automagically.
